pycti/entities/stixCyberObservables/opencti_stix_cyber_observable_latest.py

```diff
@@ -0,0 +1,2180 @@
+
+            result = self.opencti.query(
+                query,
+                {
+                    "file": (self.file(final_file_name, data, mime_type)),
+                    "x_opencti_description": x_opencti_description,
+                    "createdBy": created_by,
+                    "objectMarking": object_marking,
+                    "objectLabel": object_label,
+                },
+            )
+            return self.opencti.process_multiple_fields(
+                result["data"]["artifactImport"]
+            )
+        else:
+            self.opencti.app_logger.error("Missing parameters: type")
+
+    """
+        Update a Stix-Observable object field
+
+        :param id: the Stix-Observable id
+        :param input: the input of the field
+        :return The updated Stix-Observable object
+    """
+
+    def update_field(self, **kwargs):
+        id = kwargs.get("id", None)
+        input = kwargs.get("input", None)
+        if id is not None and input is not None:
+            self.opencti.app_logger.info("Updating Stix-Observable", {"id": id})
+            query = """
+                mutation StixCyberObservableEdit($id: ID!, $input: [EditInput]!) {
+                    stixCyberObservableEdit(id: $id) {
+                        fieldPatch(input: $input) {
+                            id
+                            standard_id
+                            entity_type
+                        }
+                    }
+                }
+            """
+            result = self.opencti.query(
+                query,
+                {
+                    "id": id,
+                    "input": input,
+                },
+            )
+            return self.opencti.process_multiple_fields(
+                result["data"]["stixCyberObservableEdit"]["fieldPatch"]
+            )
+        else:
+            self.opencti.app_logger.error(
+                "[opencti_stix_cyber_observable_update_field] Missing parameters: id and input",
+            )
+            return None
+
+    """
+        Promote a Stix-Observable to an Indicator
+
+        :param id: the Stix-Observable id
+        :return the newly created indicator
+    """
+
+    def promote_to_indicator_v2(self, **kwargs):
+        id = kwargs.get("id", None)
+        custom_attributes = kwargs.get("customAttributes", None)
+        if id is not None:
+            self.opencti.app_logger.info("Promoting Stix-Observable", {"id": id})
+            query = (
+                """
+                    mutation StixCyberObservableEdit($id: ID!) {
+                        stixCyberObservableEdit(id: $id) {
+                            promoteToIndicator {
+                                """
+                + (
+                    custom_attributes
+                    if custom_attributes is not None
+
+                    else (
+                        """
+                        id
+                        entity_type
+                        """
```

Comment:
Still WIP, I need to change that and 
1. extract the indicator properties in a file
2. import them and use it here